src/main/scala/com/typesafe/sbt/packager/rpm/RpmPlugin.scala

```diff
@@ -25,28 +25,36 @@ object RpmPlugin extends AutoPlugin {
 
   object autoImport extends RpmKeys {
     val Rpm = config("rpm") extend Linux
+    val RpmConstants = Names
   }
 
   import autoImport._
 
-  def osPostInstallMacro: java.net.URL = getClass getResource "brpJavaRepackJar"
+  private final def osPostInstallMacro: java.net.URL = getClass getResource "brpJavaRepackJar"
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [osPostInstallMacro should have a smaller scope by using private[this]](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/1559303178/issues/source?bid=2040925&fileBranchId=2489531#l33)